anchor/programs/pubkey-protocol/src/state/community.rs

```diff
@@ -22,20 +22,14 @@ pub struct Community {
     // Pending authority that must sign to complete
     pub pending_authority: Option<Pubkey>,
     // Providers (identities) user have added onto
-    pub providers: Vec<Identity>,
-    pub discord: Option<String>,
```

Comment:
We do want to keep the social links on the Community struct (related: keep `UpdateCommunityDetails` as is.)